docs/source/markdown/podman.1.md

```diff
@@ -75,110 +95,169 @@ If no identity file is provided and no user is given, podman defaults to the use
 Podman prompts for the login password on the remote server.
 
 Identity value resolution precedence:
- - command line value
- - environment variable `CONTAINER_SSHKEY`, if `CONTAINER_HOST` is found
- - `containers.conf`
-Remote connections use local containers.conf for default.
 
-#### **--log-level**=*level*
+- command line value.
+- environment variable `CONTAINER_SSHKEY`, if `CONTAINER_HOST` is found.
+- `containers.conf`.
+Remote connections use local `containers.conf` for default.
+
+#### **--log-level**=**_error_** | *debug* | *info* | *warn* | *fatal/panic*
```

Comment:
`fatal` and `panic` are two different log levels so they should be separated like the other options.

Reply:
From MANPAGE_SYNTAX:
*If the number of arguments is greater than three, the arguments are **not** listed after the equal sign. The arguments have to be shown in a table [...]*

I believe this is the better way here or the command itself will be a bit overloaded.